package-lock.json

```diff
@@ -6924,8 +6924,7 @@
         },
         "ansi-regex": {
           "version": "2.1.1",
-          "bundled": true,
-          "optional": true
+          "bundled": true
```

Comment:
Not sure if this change to the package-lock.json file is needed, as I don't see any additional dependencies or upstream changes from the scope of this PR. If these changes are required, you can ignore this comment.